index.js

```diff
@@ -10,5 +10,9 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/chat', (req, res) => {
+	res.render('chat');
```

Comment:
Indentation.

Reply:
Make it 2 spaces please.

Reply:
Ok alright.

Reply:
@sumedh123 I've made the changes you had requested. Please check.